docs/multi-worker.md

```diff
@@ -0,0 +1,133 @@
+# Mutli-Worker Fluentd Setup
+
+## Necessity
+
+In specific scenarios, a fluentd with a single worker instance cannot process and forward the high amount of logs produced on clusters. This can lead to fluentd Pods not accepting additional traffic from fluent-bits and fluent-bits suffering under Backpressure. In the end, both fluentd and fluent-bit Pods might run into their memory limits and get restarted by Kubernetes. Enabling multiple worker processes per fluentd Pod will increase the performance of this component, so it is recommended to use a multi-worker approach in environments with high log volume. Additionally the official [fluentd documentation](https://docs.fluentd.org/deployment/multi-process-workers) might be helpful.
+
+## Recommended implementation
+
+When enabling the multi-worker setup, it is recommended to ensure the following things:
+- Place the fluentds on separate Nodes
+- Increase the compute and memory resources 
+- Configure a buffer volume for fluentd
```

Comment:
nit: fluentd is configured with a buffer volume by default, I'm not entirely sure this is required (but at least it doesn't hurt either)

Reply:
You are right, I will remove it.